app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -68,6 +67,11 @@ public function index(Request $request)
                 $query = $query->where('signups.northstar_id', '=', $filters['northstar_id']);
             }
 
+            // Only return Posts for the given status (if there is one)
+            if (array_has($filters, 'status')) {
+                $query = $query->where('status', '=', $filters['status']);
```

Comment:
Oh ok, got it. So the user profile will accept any post (`pending`, `accepted`, or `rejected`) and then only the gallery needs to filter (to just `accepted`), right? That seems fine to me! 👍